packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -228,6 +240,20 @@ export class OrganizationAPI extends AbstractAPI {
     });
   }
 
+  public async getOrganizationInvitationList(params: GetOrganizationInvitationListParams) {
+    const { organizationId, status, limit, offset } = params;
+    this.requireId(organizationId);
+
+    return this.request<OrganizationInvitation[]>({
+      method: 'GET',
+      path: joinPaths(basePath, organizationId, 'invitations'),
+      queryParams: { status, limit, offset },
+    });
+  }
+
+  /**
+   * @deprecated  Use `getOrganizationInvitationList` instead along with the status parameter.
+   */
```

Comment:
Let's use the `deprecate` utility function inside the method as well. That way devs using that function can see warnings at runtime

Reply:
Added!